connection.ts

```diff
@@ -0,0 +1,181 @@
+import { BufReader, BufWriter } from "./vendor/https/deno.land/std/io/bufio.ts";
```

Comment:
Please run `deno fmt && deno lint --unstable` and fix the warnings and errors.
If it's hard to fix, please add a `deno-lint-ignore` comment to that point (like [this](https://github.com/denolib/deno-redis/blob/b4a4014585305969c8e1b58bf5409701fe3add19/redis.ts#L407)).